week3/server.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE `customer` (
```

Comment:
Comments on ER diagram:

- There should be a one to many relationship between product and cart. Every record of cart contains only one product. A better name for the cart entity would be "customer_product" or "item_in_cart", the name "cart
 makes it sound like there is more than one product in it, but in reality this table will only hold one product in every record.
- The relationship between cart and order needs improvement. You have a many to many relationship, but no junction table. You should have a junction table instead of the foreign_key cart_id. In the current setup with this foreign key, every order can only contain one cart (but if a customer is buying 20 items, the order should contain 20 carts). Again, I think the confusion comes from the naming of the entity cart.